skimage/_shared/utils.py

```diff
@@ -183,11 +405,13 @@ def _docstring_add_deprecated(func, kwarg_mapping, deprecated_version):
 
     Doc = FunctionDoc(func)
     for old_arg, new_arg in kwarg_mapping.items():
-        desc = [
-            f'Deprecated in favor of `{new_arg}`.',
-            '',
-            f'.. deprecated:: {deprecated_version}',
-        ]
+        desc = []
+        if new_arg is None:
+            desc.append(f'`{old_arg}` is deprecated.')
+        else:
+            desc.append(f'Deprecated in favor of `{new_arg}`.')
```

Comment:
```suggestion
            desc.append(f'{old_arg}` is deprecated in favor of `{new_arg}`.')
```

Reply:
This description directly follows the name of `old_arg: DEPRECATED \n`. I'm a bit to lazy to update the tests so that this message passes unless there's a good reason to include the name again?

Reply:
Oh, fine then. Please ignore my suggestion.